services/py-api/src/server/handlers/participants_handlers.py

```diff
@@ -46,6 +63,10 @@ async def create_participant(
             if isinstance(result.err_value, HackathonCapacityExceededError):
                 response.status_code = status.HTTP_409_CONFLICT
                 return ErrResponse(error="Max hackathon capacity has been reached")
+            
+            if isinstance(result.err_value, TeamCapacityExceededError):
+                response.status_code = status.HTTP_409_CONFLICT
+                return ErrResponse(error="Your team's capacity has been reached")
```

Comment:
Change the comment to "Max team capacity has been reached to match the same structure as the error above.

Reply:
Fixed in commit a9a6b60a1c533cba63bd89fc5bee4362218c00f7